src/layouts/DarkMode.astro

```diff
@@ -0,0 +1,93 @@
+---
+const isDarkMode = new Date().getHours() > 18 || new Date().getHours() < 6;
+---
+
+
+  <head>
+    <style>
+    :root {
+      --background-color: white;
+      --text-color: black;
+      --toggle-background: #4c4c4c;
+      --toggle-color: #b3b3b3;
+      --toggle-circle: #9b59b6;
+    }
+
+    .toggle-container {
+      display: flex;
+      align-items: center;
+      justify-content: center;
+  
+    }
+
+    .toggle-button {
+      width: 70px;
+      height: 38px;
+      background-color: var(--toggle-background);
+      border-radius: 25px;
+      position: relative;
+      cursor: pointer;
+      transition: background-color 1s, color 1s;
+      border: 1px solid #fff;
+    }
+
+    .toggle-circle {
+      width: 37px;
+      height: 37px;
+      background-color: var(--toggle-circle);
+      border-radius: 50%;
+      position: absolute;
+      top: 1px;
+      /* left: 8px; */
+      transition: transform 0.3s;
+    }
+
+    .toggle-button.dark .toggle-circle {
+      transform: translateX(33px);
+    }
+
+    .icon {
+      width: 14px;
+      height: 14px;
+      position: absolute;
+      top: 50%;
+      transform: translateY(-50%);
+    }
+
+    .icon.sun {
+      left: 12px;
+    }
+
+    .icon.moon {
+      right: 12px;
+    }
+      </style>
+  </head>
+  <body class={isDarkMode ? "dark" : ""}>
```

Comment:
no body should be here since it is a component

Reply:
Added in component?

Reply:
Oh, yes correct in component